Create Pull Request Template (#2755)
highlightjs/highlight.js: .github/pull_request_template.md

<!--- Provide a general summary of your changes in the Title above -->

> <!-- Please link to a related issue below. -->

Reviewer: ```suggestion
<!-- Please link to a related issue below. -->
<!-- ie, `Resolves #1234`, etc... so GitHub can magically link it -->
```

> ## Changes
> <!--- Describe your changes -->
> 
> ## Checklist

Reviewer: ```suggestion
### Changes
<!--- Describe your changes -->

### Checklist
```

A bit too bold for my taste.

> - [ ] I added markup tests, or they don't apply here because...
> - [ ] I updated the changelog at `CHANGES.md`
> - [ ] My name is in `AUTHORS.txt`, under contributors

Reviewer: ```suggestion
- [ ] Added markup tests, or they don't apply here because this is a minor keyword bump...
- [ ] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors
```

Author: I left it blank after "because", so that the author would be forced to specify the reason. What do you think?

Reviewer: Ok... `because... ` then.

Author: ```suggestion
- [ ] Added markup tests, or they don't apply here because...
- [ ] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors
```
